### ISSUE TITLE
No colours in notifier

### ISSUE THREAD

**Issue by iainbeeston** [alloy/kicker#53]
I use kicker with the terminal-notifier gem, so that when each test run completes I get a native OS X notification of the result. Unfortunately, the notification text includes special characters for colouring the test output:

![screen shot 2014-11-07 at 15 58 50](https://cloud.githubusercontent.com/assets/111963/4955755/8fd735d4-6697-11e4-9c93-8685324db0fd.png)

I'm not sure if this applies equally to all notifier gems, but should special characters like these be stripped before being sent to the notifier?

**Comment by Manfred** [comment]
Yes, it should be stripped before it's being passed to the notifier. I think this is the place where it should happen:

https://github.com/alloy/kicker/blob/master/lib/kicker/job.rb#L49
